<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterButtonForm</class>
 <widget class="QWidget" name="FilterButtonForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>158</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>251</width>
    <height>158</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>251</width>
    <height>158</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox QAbstractItemView {
   background-color: rgb(39, 39, 49);
   selection-background-color: rgb(60, 60, 72);
   color:white;
   border: 1px solid rgb(88, 88, 100);
   border-radius: 4px;
   padding: 4px;
   outline: 0;
}
QComboBox{	
border: 1px solid rgb(184,191,200);
background-color: rgb(39, 39, 49);
color: white;
}

QWidget{
background-color: rgb(31,31,31);
}
QLineEdit{
border: 1px solid rgb(184,191,200);
color: rgb(184,191,200);
}
QLabel{
color: rgb(184,191,200);
QLabel::setAlignment (Qt::AlignVCenter );
QLabel::setAlignment (Qt::AlignHCenter );
}
QPushButton{
color: rgb(184,191,200);
background-color: rgb(10, 21, 49);
border: 1px solid rgb(184,191,200);
border-radius: 3px;
}
QPushButton:hover
{
border: 2px solid red;
background-color: rgb(103, 104, 114);
}
QPushButton:pressed
{
background-color: rgb(35,190,212);
color: rgb(255, 255, 255);
}

</string>
  </property>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sortComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Choose Filter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Median</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Low Pass</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>High Pass</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>10</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Form Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="sortLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter Sort</string>
   </property>
  </widget>
  <widget class="QLabel" name="valueLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="placeholderText">
    <string>Choose Filter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>FilterButtonForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
